app/views/pages/home.html.erb

```diff
@@ -1,2 +1,4 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<div class = "container" style="min-height:500px">
```

Comment:
put in a scss

Reply:
changed scss. please review it and approve !